cli/waiter/util.py

```diff
@@ -117,3 +117,9 @@ def load_json_file(path):
         logging.info(f'{path} is not a file')
 
     return content
+
+
+def is_service_current(service, current_token_etag):
```

Comment:
Should this method check both the token and the version? Two different tokens can have the same etag.